hw/BUILD

```diff
@@ -30,7 +30,6 @@ fusesoc_build(
         "//:cores",
     ],
     data = ["//hw/ip/otbn:all_files"],
-    flags = ["--verilator_options=--threads 1"],
```

Comment:
Ah nice! Thanks for finding this. Is it now building the Verilator model to run on 4 threads? If so, is it possible to change it to run with 1 thread in CI? We run 4 parallel models simultaneously in CI, expecting each model to take 1 thread.

Reply:
Here's where it's set in CI: https://cs.opensource.google/opentitan/opentitan/+/master:ci/scripts/run-verilator-tests.sh;drc=3f7956ee10c794d7327483d1172d22e80937ec72;l=17 I think, there's some unexercised code here that had me confused: https://cs.opensource.google/opentitan/opentitan/+/master:ci/scripts/build-chip-verilator.sh

Reply:
Ah I see -- okay looks good then.